pkg/config/config.go

```diff
@@ -51,58 +51,68 @@ func Error() error {
 
-	// Disable is use in Prometheus to disable v3io and work with the internal TSDB
+	// Disabled = true disables the V3IO TSDB configuration in Prometheus and
+	// enables the internal Prometheus TSDB instead
 	Disabled bool `json:"disabled,omitempty"`
-	// Set logging level: debug | info | warn | error (info by default)
+	// Logging verbosity level - "debug" | "info" | "warn" | "error"
 	Verbose string `json:"verbose,omitempty"`
 	// Number of parallel V3IO worker routines
 	Workers int `json:"workers"`
-	// Number of parallel V3IO worker routines for queries (default is min between 8 and Workers)
+	// Number of parallel V3IO worker routines for queries;
+	// default = the minimum value between 8 and Workers
 	QryWorkers int `json:"qryWorkers"`
 	// Max uncommitted (delayed) samples allowed per metric
 	MaxBehind int `json:"maxBehind"`
-	// Override last chunk (by default on restart it will append from the last point if possible)
+	// Override last chunk; by default, an append from the last point is attempted upon restart
 	OverrideOld bool `json:"overrideOld"`
-	// Default timeout duration in Seconds (if not set, 1 Hour timeout will be used )
+	// Default timeout duration, in seconds; default = 3,600 seconds (1 hour)
 	DefaultTimeoutInSeconds int `json:"timeout,omitempty"`
-	// The size of batch to use during ingestion
+	// Size of the samples batch to use during ingestion
 	BatchSize int `json:"batchSize,omitempty"`
-	// Sample size in bytes in worst compression scenario
+	// Maximum sample size, in bytes (for the worst compression scenario)
 	MaximumSampleSize int `json:"maximumSampleSize,omitempty"`
-	// Max size of a partition object
+	// Maximum size of a partition object
 	MaximumPartitionSize int `json:"maximumPartitionSize,omitempty"`
-	// Size of chunk in bytes for worst an best compression scenarios
+	// Minimum chunk size, in bytes (for the best compression scenario)
 	MinimumChunkSize int `json:"minimumChunkSize,omitempty"`
+	// Maximum chunk size, in bytes (for the worst compression scenario)
 	MaximumChunkSize int `json:"maximumChunkSize,omitempty"`
-
-	// Metrics reporter configuration
+	// Metrics-reporter configuration
 	MetricsReporter MetricsReporterConfig `json:"performance,omitempty"`
-	// dont aggregate from raw chuncks, for use when working as Prometheus TSDB lib
+	// Don't aggregate from raw chunks, for use when working as a Prometheus
+	// TSDB library
 	DisableClientAggr bool `json:"disableClientAggr,omitempty"`
 }
 
 type MetricsReporterConfig struct {
-	ReportOnShutdown   bool   `json:"reportOnShutdown,omitempty"`
-	Output             string `json:"output"` // stdout, stderr, syslog, etc.
-	ReportPeriodically bool   `json:"reportPeriodically,omitempty"`
-	RepotInterval      int    `json:"reportInterval"` // interval between consequence reports (in Seconds)
+	// Report on shutdown (Boolean)
+	ReportOnShutdown bool `json:"reportOnShutdown,omitempty"`
+	// Output destination - "stdout", "stderr", "syslog", etc.
```

Comment:
please remove `, "syslog", etc.` as unsupported

Reply:
Done (although "syslog, etc." was in the original comment :-)).